### PR TITLE
sample app: implement same icon style like miniapp list in other places

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
- MiniApp list is showing full icon instead of cropping in a circle, this PR aims to adopt the same consistency in other places as well.
- The UX of item in both regular miniapp & downloaded miniapp list can be same.

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

File: testapp/src/main/res/layout/downloaded_item_list_miniapp.xml [hunk 1]
```diff
@@ -1,26 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout>
 
-  <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (khairul-alam-licon) [reply testapp/src/main/res/layout/downloaded_item_list_miniapp.xml:4]: Yeah, yesterday I was trying to use `ConstraintLayout`, the main goal is to align the internal views horizontally. Let me to try..

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListAdapter.kt [hunk 2]
```diff
@@ -24,7 +23,7 @@ class MiniAppDownloadedListAdapter(private val miniAppListener: MiniAppDownloade
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        setIcon(holder.miniAppRoot.context, Uri.parse(miniApps[position].icon), holder.miniAppIcon)
+        holder.miniAppIcon.load(holder.miniAppRoot.context, miniApps[position].icon)
```
[the remark this answers is not in the document]
Comment (khairul-alam-licon) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListAdapter.kt:26]: Nah, Minh has added [the placeholder default icon](https://github.com/rakutentech/android-miniapp/blob/98f05a4de9b5940ad567e4360aeee778b8b191a0/testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/AppHelper.kt#L74) in `load`, so we are not dropping it still. Only thing we are dropping is `circleCrop()`.

[the remark this answers is not in the document]
Comment (Climbatize) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/permission/MiniAppDownloadedListAdapter.kt:26]: Oh I see, good job then

**Review by corycaywood — APPROVED** [review]
LGTM 👍

**Pull request merged**